app/views/dashboard/show.html.erb

```diff
@@ -38,7 +38,7 @@
       <li>Use Ruby on Rails</li>
     </ul>
 
-    <h4><%= latest_event.start_date.to_s(:long) %> - It begins!</h4>
```

Comment:
Lets use the `local_time` helper so it shows in the user's brower timezone which will be most intuitive for users.

Reply:
great idea, done!